File: DotNet/VS2003/Windows/Other/Microsoft Enterprise Instrumentation Framework/Simple Instrumentation/cs/SimpleInstrumentation.cs

// Copyright (c) Microsoft Corporation.  All rights reserved.

using System;
using System.ComponentModel;
using Microsoft.EnterpriseInstrumentation;
using Microsoft.EnterpriseInstrumentation.Schema;

namespace SimpleInstrumentation
{
	[RunInstaller(true)]
	public class MyProjectInstaller : ProjectInstaller {}

	/// <summary>
	/// A simple example of raising an event.
	/// </summary>
	public class HelloWorld
	{
		static void Main()
		{
			// Raise a TraceMessageEvent event through the Application EventSource.
			TraceMessageEvent.Raise("Hello World");
		}
	}
}

File: DotNet/VS2003/Windows/Other/Microsoft Enterprise Instrumentation Framework/Simple Instrumentation/vb/SimpleInstrumentation.vb

' Copyright (c) Microsoft Corporation.  All rights reserved.

Imports System
Imports System.ComponentModel
Imports Microsoft.EnterpriseInstrumentation
Imports Microsoft.EnterpriseInstrumentation.Schema


Namespace SimpleInstrumentation
    <RunInstaller(True)> Public Class MyProjectInstaller
        Inherits ProjectInstaller
    End Class

    ' <summary>
    ' A simple example of raising an event.
    ' </summary>
    Public Class HelloWorld
        Public Shared Sub Main()
            ' Raise a TraceMessageEvent event through the Application EventSource.
            TraceMessageEvent.Raise("Hello World")
        End Sub
    End Class
End Namespace
